ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let firstNameLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    private var avatarView: AvatarView
+    private var firstNameLabel: UILabel?
+    private var lastNameLabel: UILabel?
+    private var identifierLabel: UILabel?
+
+    /// Initializes the contact view with an avatar view, first name, and last name
+    ///
+    /// - Parameters:
+    ///   - avatarView: The AvatarView that will be displayed within the ContactView
+    ///   - firstName: String that will be the text of the top label
+    ///   - lastName: String that will be the text of the bottom label
+    public init(avatarView: AvatarView, firstName: String, lastName: String) {
+        self.avatarView = avatarView
+        self.firstNameLabel = UILabel()
+        self.lastNameLabel = UILabel()
+        super.init(frame: .zero)
+        self.backgroundColor = Colors.surfacePrimary
+        self.translatesAutoresizingMaskIntoConstraints = false
+        setupFirstNameLabel(using: firstName)
+        setupLastNameLabel(using: lastName)
+        setupLayout()
+    }
+
+    /// Initializes the contact view with an avatar view and an identifier
+    ///
+    /// - Parameters:
+    ///   - avatarView: The AvatarView that will be displayed within the ContactView
+    ///   - identifier: String that will be used to identify the contact (e.g. email, phone number, first name)
+    public init(avatarView: AvatarView, identifier: String) {
+        self.avatarView = avatarView
+        super.init(frame: .zero)
+        self.backgroundColor = Colors.surfacePrimary
```

Comment:
can we make a private initializer that both public initializers call into? That way we can share some of this code, and future updates will be less prone to bugs

Reply:
Great suggestion! I took a look into `convenience` and `designated` initializers for this and will be pushing it soon.